manim/mobject/matrix.py

```diff
@@ -81,9 +107,36 @@ def __init__(
         **kwargs,
     ):
         """
-        Matrix can either either include numbers, tex_strings,
-        or mobjects
+
+        Parameters
+        ----------
+        matrix : np.array|List[List]
```

Comment:
Suggestion : You should use typing.Iterable instead IMO

Reply:
good point again. thanks. fixing it now